docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -283,3 +283,54 @@ GET /transactions/{tx_id}
 
    :statuscode 200: A transaction with that ID was found.
    :statuscode 404: A transaction with that ID was not found.
+
+
+GET /unspents/
```

Comment:
It's consistent with out intention to rename the method `get_owned_ids()` to `get_unspents()`, as per issue #603